File: URI_1116.cpp
#include <bits/stdc++.h>
 
using namespace std;
 
int main() {
 
        int N;
        cin>>N;
		int x,y,i;
		double result;
		
		for ( i = 0; i < N; i++) {
			
			cin>>x;
			cin>>y;
			
			if ( y==0 ) {
				cout<<"divisao impossivel"<<endl;
			}else {
				result = ((double)x) / ((double)y);
				printf("%.1lf\n",result);
			}
			
		}
 
    return 0;
}

File: URI_1116.java
import java.io.IOException;
import java.util.Scanner; 
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scanner = new Scanner(System.in);
		int N = scanner.nextInt();
		int x,y;
		double result;
		
		for (int i = 0; i < N; i++) {
			
			x = scanner.nextInt();
			y = scanner.nextInt();
			
			if ( y==0 ) {
				System.out.println("divisao impossivel");
			}else {
				result = ((double)x) / ((double)y);
				System.out.printf("%.1f\n",result);
			}
			
		}
 
    }
 
}
